js/ui/appDisplay.js

```diff
@@ -2822,19 +2823,60 @@ class HackAppIcon extends AppIcon {
         });
     }
 
+    _checkNewRelease() {
+        this._flatpak_link.query_info_async(Gio.FILE_ATTRIBUTE_TIME_CHANGED,
+                                            0, 0, null, (file, res) => {
+            let info = file.query_info_finish(res);
+            let flatpak = info.get_attribute_uint64(Gio.FILE_ATTRIBUTE_TIME_CHANGED);
+            let last_launch = global.settings.get_int64('hack-last-launch');
```

Comment:
Let's start using `const` instead of `let` for variables that are only assigned once.